### PR TITLE
Split of the camera documentation

### PR DESCRIPTION
**Pull request by naushir**
- Branch to split the camera documentation
- Restructure camera documentation
- camera: Update maximum exposure times
- camera: Update sensor module table
- camera: Remove "Software Features" section
- camera: Minor updates to the camera guide

### CODE REVIEW

**Comment by naushir** [comment]
@aallan here's a first pass at moving the camera documentation around.

File: documentation/asciidoc/computers/camera/camera_usage.adoc [hunk 1]
```diff
@@ -4,13 +4,13 @@ Raspberry Pi currently sell two types of camera board: an https://www.raspberryp
...
 
 [NOTE]
 ====
-Raspberry Pi is transitioning from a legacy camera software stack based on proprietary Broadcom GPU code to an open-source stack based on `libcamera`. Raspberry Pi OS images from _Bullseye_ onwards will contain *only* the `libcamera`-based stack. Raspberry Pi OS images up to and including _Buster_ will contain the legacy _Raspicam_ stack, though the `libcamera` stack and applications can be installed xref:camera.adoc#libcamera-and-libcamera-apps-packages[using _apt_], or built by following the xref:camera.adoc#building-libcamera-and-libcamera-apps[normal build instructions].
+Raspberry Pi has transitioned from a legacy camera software stack based on proprietary Broadcom GPU code to an open-source stack based on `libcamera`. Raspberry Pi OS images from _Bullseye_ onwards will contain *only* the `libcamera`-based stack. Raspberry Pi OS images up to and including _Buster_ will contain the legacy _Raspicam_ stack, though the `libcamera` stack and applications can be installed xref:camera.adoc#libcamera-and-libcamera-apps-packages[using _apt_], or built by following the xref:camera.adoc#building-libcamera-and-libcamera-apps[normal build instructions].
```
[the remark this answers is not in the document]
Comment (lurch) [reply documentation/asciidoc/computers/camera/camera_usage.adoc:13]: > will be removed in a future version of Raspberry Pi OS

It has _already_ been removed in the current version of Raspberry Pi OS: "Raspberry Pi OS images from _Bullseye_ onwards will contain *only* the `libcamera`-based stack".

[the remark this answers is not in the document]
Comment (aallan) [reply documentation/asciidoc/computers/camera/camera_usage.adoc:13]: > Probably this should get more extensive rewording. Perhaps also an admonition block, maybe something like:
> 
> NOTE: The lgacy firmware-based camera stack, including `raspicam`, has been deprecated and will be removed in a future version of Raspberry Pi OS. To use the legacy camera stack, enable it using the `raspi-config` utility: select `legacy camera` under `interface options`.

Hey. We're not really looking for external feedback on this branch at this time. Sorry.

[the remark this answers is not in the document]
Comment (naushir) [reply documentation/asciidoc/computers/camera/camera_usage.adoc:13]: > Probably this should get more extensive rewording

TBH, I don't really know what more I would change in the above paragraph.  Suggestions?

**Comment by aallan** [comment]
> As we now have both `documentation/asciidoc/accessories/camera.adoc` and `documentation/asciidoc/computers/camera.adoc` I wonder if it _might_ make sense to rename the latter to `documentation/asciidoc/computers/camera_software.adoc` ? 🤷

Maybe. The file name isn't really exposed in the text though, so it's "just" a backend thing that we can do at any point.

**Comment by naushir** [comment]
> As we now have both `documentation/asciidoc/accessories/camera.adoc` and `documentation/asciidoc/computers/camera.adoc` I wonder if it _might_ make sense to rename the latter to `documentation/asciidoc/computers/camera_software.adoc` ? 🤷

Sure, I can do that change.

File: documentation/asciidoc/computers/camera/camera_usage.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrum99) [reply documentation/asciidoc/computers/camera/camera_usage.adoc:13]: > > Probably this should get more extensive rewording. Perhaps also an admonition block, maybe something like:
> > NOTE: The lgacy firmware-based camera stack, including `raspicam`, has been deprecated and will be removed in a future version of Raspberry Pi OS. To use the legacy camera stack, enable it using the `raspi-config` utility: select `legacy camera` under `interface options`.
> 
> Hey. We're not really looking for external feedback on this branch at this time. Sorry.

Not a problem. As you were.

**Comment by aallan** [comment]
Thanks for this @naushir. Just getting out of bed here in California, so I'll take a proper look at this after breakfast. Potentially we do minimal changes in this initial split, merge this in, and then the rewriting happens on a new branch after the merge?

**Comment by aallan** [comment]
Hey @naushir. You did your PR from your fork of the repo onto `develop` rather than the `camera_branch` so I've changed the base branch for the PR.

**Comment by aallan** [comment]
I'll now merge this onto the `camera_branch` and we can take a look on that branch how things look, work on branch, and then merge the `camera_branch` back to `develop` when we're ready to go, and hence to `master` and onto the live site?

**Comment by aallan** [comment]
This looks like a good first cut to me. 

Looking at it, I think @lurch is right and the `camera.adoc` file in Computers should probably get a rename, to `camera_software.adoc` to make `xref:`'s across the pages cleaner.

We also need some linking text between the two pages, and the `raspicam` stuff needs to get moved to the end and get a big `DEPRECATED` warning box stuck on it.

How much new material do you think there is to go in @naushir ?

**Comment by naushir** [comment]
> This looks like a good first cut to me.
> 
> Looking at it, I think @lurch is right and the `camera.adoc` file in Computers should probably get a rename, to `camera_software.adoc` to make `xref:`'s across the pages cleaner.

Sure, I'll make that change.

> 
> We also need some linking text between the two pages, and the `raspicam` stuff needs to get moved to the end and get a big `DEPRECATED` warning box stuck on it.

There is a cross link between pages ("Preparing the SW" from the accessories page, and "Once installed..." from the camera software page).  Perhaps this need to be more explicit?

I'll also move the Raspicam stuff right to the end of the document.

> 
> How much new material do you think there is to go in @naushir ?

Nothing more for now... I'll look though again, but I think I've made all the amendments required.

**Comment by naushir** [comment]
@aallan and @lurch I've done something a bit silly unfortunately.

I failed to see that this PR had already been merged into https://github.com/raspberrypi/documentation/tree/camera_branch. and I've gone ahead and made the above suggestions (including a bunch of file renames) in my branch at https://github.com/naushir/documentation/tree/camera. This overwrites changes in the existing PR :-(

We could either force push the changes to camera_branch, or (preferably) removing the existing camera_branch and I create a new PR for camera_branch with my latest set of changes.  What do you think?

**Comment by aallan** [comment]
I'll leave it to @lurch to decide what's the best approach. His Git skills are superior to mine. Andrew?

**Comment by naushir** [comment]
One option would be for either of you to create a new branch on here called camera_branch_v2, with the tip at 7b621a46fd82576f795efca3fc8872ba07cde9b1.  I can then create a PR with all my changes for camera_branch_v2 as normal, and we abandon camera_branch.

**Comment by naushir** [comment]
If that is too troublesome, I can redo my work and base it on top of camera_branch.  Let me know.

**Comment by lurch** [comment]
> One option would be for either of you to create a new branch on here called `camera_branch_v2`, with the tip at https://github.com/raspberrypi/documentation/commit/7b621a46fd82576f795efca3fc8872ba07cde9b1.

I've (hopefully) just done this - let me know if it doesn't look right!

**Comment by naushir** [comment]
Great, I'll push the PR now...

**Comment by naushir** [comment]
See new PR at https://github.com/raspberrypi/documentation/pull/2673

**Comment by aallan** [comment]
Closing in favour of #2673.

**Pull request merged**